src/app/landing/Testimonial.jsx

```diff
@@ -1,14 +1,51 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './Testimonial.css';
 
-const Testimonial = ({ name, img, text }) => (
+const Testimonial = ({ imagePartialPath, name, text }) => (
   <div className="Testimonial">
-    <img src={img} className="Testimonial-image" alt={`${name}'s face`} />
+    <picture>
+      {/* desktop jpg */}
+      <source
+        media="(min-width: 1224px)"
+        srcSet={`/img/landing_page/${imagePartialPath}_desktop1x.jpg`}
+        type="image/jpeg"
+      />
+      {/* desktop webp */}
+      <source
+        media="(min-width: 1224px)"
+        srcSet={`/img/landing_page/${imagePartialPath}_desktop1x.webp`}
+        type="image/webp"
+      />
+      {/* mobile jpg */}
+      <source
+        srcSet={`/img/landing_page/${imagePartialPath}_mobile2x.jpg`}
+        type="image/jpeg"
+      />
+      {/* mobile webp */}
+      <source
+        srcSet={`/img/landing_page/${imagePartialPath}_mobile2x.webp`}
+        type="image/webp"
+      />
+      <img
+        className="Testimonial-image"
+        src={`/img/landing_page/${imagePartialPath}_mobile2x.jpg`}
+        alt={`${name}'s face`}
+      />
+    </picture>
     <div className="Testimonial-separator" />
-
     <div className="Testimonial-text">“{text}”</div>
     <div className="Testimonial-name">– {name}</div>
   </div>
 );
 
+Testimonial.propTypes = {
+  name: PropTypes.string.isRequired,
+  img: PropTypes.shape({
```

Comment:
This propType doesn't match the props actually being used.

Reply:
Good catch, fixed both in Testimonial and PartnerPicture component